conda/environments/cuml_dev_cuda11.0.yml

```diff
@@ -12,9 +12,9 @@ dependencies:
 - cudf=21.08.*
 - rmm=21.08.*
 - libcumlprims=21.08.*
-- dask-cudf=21.081.08.*
+- dask-cudf=21.08.08.*
```

Comment:
Is it a real version?  `21.08.08` vs `21.08`.

Reply:
dang the bulk find and replace, didn't notice that happened, thanks for noticing!